scripts/ci/fetch.github.sh

```diff
@@ -22,10 +22,8 @@
 set -e
 
 # install dependencies
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv B00A0BD1E2C63C11
-echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu jammy/mongodb-org/5.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb.list
```

Comment:
Why was the ubuntu package for mongo removed?

Reply:
After some internal discussion it's not more needed to install this mong package